eagleproject/core/blockchain.py

```diff
@@ -241,26 +241,23 @@ def strategyCheckBalance(strategy, coin_contract, decimals, block="latest"):
         return Decimal(0)
 
 
-def rebasing_credits_per_token(block):
-    data = storage_at(OUSD, 59, block)
+def rebasing_credits_per_token(block="latest"):
+    signature = "0x6691cb3d"  # rebasingCreditsPerToken()
+    data = call(OUSD, signature, "", block)
     return Decimal(int(data["result"][0 : 64 + 2], 16)) / Decimal(math.pow(10, 18))
 
 
-def ousd_rebasing_credits(block):
-    data = storage_at(OUSD, 58, block)
+def ousd_rebasing_credits(block="latest"):
+    signature = "0x077f22b7"  # rebasingCredits()
+    data = call(OUSD, signature, "", block)
     return Decimal(int(data["result"][0 : 64 + 2], 16)) / Decimal(math.pow(10, 18))
 
 
-def ousd_non_rebasing_credits(block):
```

Comment:
ousd_non_rebasing_credits is basically a meaningless number. We just tracked because the old contract did. It's not in the new contract.